lxd/daemon.go

```diff
@@ -1233,6 +1234,16 @@ func (d *Daemon) init() error {
 		version.UserAgentFeatures([]string{"cluster"})
 	}
 
+	// Setup the UUID file.
+	// Use os.VarDir to allow setting up the uuid file also in the test suite.
+	uuidPath := filepath.Join(d.os.VarDir, "uuid")
```

Comment:
Can we call this server.uuid so it aligns with server.crt and server.key please

Reply:
This should also be loaded here into daemon such that its returned as a field in d.State().

Reply:
Now loaded from `State`